db_schema.md

```diff
@@ -86,3 +89,18 @@
 - user_id (integer, foreign key to user table)
 - user (auto reference to the User object that owns this Log)
 - icon (string, the icon to use for the bullet point in the notebook)
+
+### simulations_users (to track which simulations the user does and which users do which simulations)
```

Comment:
Have we talked through these three new proposed tables? I would prefer to not add them at this time, and stick with just the API addition to enable the fancy new js frontend for the students.